docs/src/doc/create-reusable-components/03_developing_components.stories.mdx

```diff
@@ -5,14 +5,38 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Developing Components
 
+Due to the decentralized nature of the design system, developers embedded in different feature teams will be accountable for creating new components to use in the feature they are building.
```

Comment:
This whole page is really great! thank you for the documentation. The only thing I would add here is maybe a section describing the step by step workflow for a component as a quick reference and overview. Something like:

0. Gather relevant alias tokens
0. Create component tokens
0. Write component code (and tests)
0. Create e2e tests (regression, selenium)

etc...

Reply:
That's a good idea, will do.

Reply:
That sounds good! Outlining the workflow in both the design and tech sections in "How to create reusable components" will make writing the Overview easier (we could _simply_ collect all the steps with a short description in there).

Reply:
perfect! looking great, thank you :)